src/resources/js/dialog.js

```diff
@@ -1,17 +1,17 @@
 var tribe = tribe || {};
 tribe.dialogs = tribe.dialogs || {};
-tribe.dialogs.dialogs = tribe.dialogs.dialogs || {};
+tribe.dialogs.dialogs = tribe.dialogs.dialogs || [];
```

Comment:
We should move the two definitions for `tribe.dialogs.dialogs` and `tribe.dialogs.events` to inside of the "module".